README.md

```diff
@@ -1,13 +1,25 @@
-# lium - Abstraction Layer of ChromiumOS development
+# lium - Make ChromiumOS development extreamely easy
 
-`lium` is an abstruction layer of ChromiumOS development environment and workflows.
+`lium` is an extreamely user-friendly tool for ChromiumOS developers.
 
-It provides a simple interface for common tasks for ChromiumOS developers,
-with plenty of automatic error recovery mechanisms to avoid keep sticking your eyes on the display(s),
-give you some time for a nap and/or coffee, or other tasks ;)
+It provides a simple way to do common development tasks and make the bareer for contributing to ChromiumOS and lium itself as low as possible.
 
 Also, it manages local development hardware including DUTs and Servos, and act as working examples of commands to interact with them.
 
+We hope lium gives you some time for a nap and/or coffee, or other tasks by making your work more effective ;)
+
+## Core principles
+
+- P0: Make the basic ChromiumOS development workflow extremely easy
+  - Background: There have been a huge barriers to get started with the ChromiumOS development. It terrifies newcomers sometimes, and such environment is not sustainable or efficient. The top priority of lium is to mitigate this high barrier to enter.
+  - Basic workflows include: checkout the source code, build images / deploying packages / run tests with / without modifications.
+- P1: Make it extreamely easy to start using lium
+  - Background: As stated in the above, our goal is lowering the barrier for people who are about to start contribution to ChromiumOS. To achive that, a tool that aids the goal should be extremely easy as well to start using it.
+  - How: Follow best practices and common ways to do things. Prefer defaults always if there is no clear reason to change that.
+- P2: Be a working example compiled from knowledge in the documentations
+  - Background: Documentations can be rotten sirently. Code rots as well, but it will stop working sooner. People often prefer coding than documentation.
+  - How: Put enough background information in the code as comments. Put links to the documentations. Put anything useful with code, and wrap them with logics. It gives better understanding on the things than only using natural languages.
```

Comment:
nit: Consistent subject

Either change Put enough to It puts enough or change It gives to Give